authentication/authentication.go

```diff
@@ -0,0 +1,153 @@
+package authentication
+
+import (
+	"crypto/rsa"
+	"crypto/sha1"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"gopkg.in/square/go-jose.v2"
+	"gopkg.in/square/go-jose.v2/jwt"
+	"io/ioutil"
+)
+
+// wraps raw data in JWT and JWE
```

Comment:
Comments on exported functions should start with the name of the function:
```
// GetJwe is a thing that does a thing
func GetJwe( ... ) { ... }
```

Reply:
Done